ccm_web/server/src/api/api.controller.ts

```diff
@@ -139,10 +140,9 @@ export class APIController {
   @Get('admin/user/:loginId')
   async getUserInfoAsAdmin (
     @Param('loginId') loginId: string
-  ): Promise<CanvasUser> {
+  ): Promise<void> {
     const result = await this.apiService.getUserInfoAsAdmin(loginId)
     if (isAPIErrorData(result)) throw new HttpException(result, result.statusCode)
-    return result
```

Comment:
If this is only used by `checkIfUserExists` in UI code and will not return any value, I suggest renaming the method.  `checkIfUserExists` or `userExists` would be better.  To me, the `get` prefix of the name indicates that it will return  some data.

Reply:
Okay, how is this? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/363/commits/a72d0d017ebdc4a3985b59d809039ea19c23271e